tripal_chado/tests/src/Kernel/ChadoCustomTables/ChadoMviewTest.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\tripal_chado\Kernel\ChadoCustomTables;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Tests\tripal_chado\Kernel\ChadoTestKernelBase;
+use Drupal\tripal\Services\TripalLogger;
+use Drupal\tripal_chado\Database\ChadoConnection;
+use Drupal\tripal_chado\ChadoCustomTables\ChadoMview;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
+use PHPUnit\Framework\Attributes\Test;
+
+/**
+ * Kernel test for ChadoMview create/populate/delete lifecycle.
+ *
+ * @group tripal-chado
+ * @group service-chado-custom-table
+ */
+#[CoversClass(ChadoMview::class)]
+#[Group('tripal-chado')]
+#[Group('service-chado-custom-table')]
```

Comment:
We no longer use the tripal-chado group. We could benefit from an mview group though 🤔 


```suggestion
#[Group('service-chado-mview')]
#[Group('service-chado-custom-table')]
```